cmd/treedb_text_hybrid_scale/main_test.go

```
@@ -159,6 +239,172 @@ func TestRankBottlenecksNormalizesUnits2731(t *testing.T) {
+		"-query-rows", queryRowHybridTextScalar,
+		"-cpu-profile", alloc + ".base",
+		"-alloc-profile", alloc,
+	})
+	if err == nil || !strings.Contains(err.Error(), "generated -alloc-profile baseline") || !strings.Contains(err.Error(), "must not resolve to the same path") {
+		t.Fatalf("generated baseline collision error=%v", err)
+	}
+}
+
+func TestTenMPlanPropagatesPhaseSelector4327(t *testing.T) {
+	runDir := t.TempDir()
+	cmd := exec.Command("bash", "scripts/bench_text_hybrid_scale.sh")
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)